docs/.vuepress/components/Testimonial.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
+        <h1></h1>
```

Comment:
If it is for more space, we should add some margin on the div above instead, WDYT ?